CanaryValidator/README.md

```diff
@@ -7,26 +7,36 @@ Canary can be invoked either as Service Administrator or Tenant Administrator.
 
 # Download Canary
 ```powershell
-invoke-webrequest https://github.com/Azure/AzureStack-Tools/archive/master.zip -OutFile master.zip
-expand-archive master.zip -DestinationPath . -Force
-cd AzureStack-Tools-master\CanaryValidator
+Invoke-WebRequest https://github.com/Azure/AzureStack-Tools/archive/master.zip -OutFile master.zip
+Expand-Archive master.zip -DestinationPath . -Force
+Set-Location -Path ".\AzureStack-Tools-master\CanaryValidator" -PassThru
 ```
 
-# To execute Canary as Tenant Administrator
+# To execute Canary as Tenant Administrator (if Windows Server 2016 or Windows Server 2012-R2 images are already present in the PIR)
 ```powershell
 # Install-Module AzureRM -RequiredVersion 1.2.8 -Force
 # Install-Module AzureStack
 $TenantAdminCreds =  New-Object System.Management.Automation.PSCredential "<Tenant Admin username>", (ConvertTo-SecureString "<Tenant Admin password>" -AsPlainText -Force)
 $ServiceAdminCreds =  New-Object System.Management.Automation.PSCredential "<Service Admin username>", (ConvertTo-SecureString "<Service Admin password>" -AsPlainText -Force)
-.\Canary.Tests.ps1  -AADTenantID "<TenantID from Azure Active Directory>"  -AdminArmEndpoint "<Administrative ARM endpoint>" -ServiceAdminCredentials $ServiceAdminCreds -TenantArmEndpoint "<Tenant ARM endpoint>" -TenantAdminCredentials $TenantAdminCreds
+.\Canary.Tests.ps1  -TenantID "<TenantID from Azure Active Directory>" -AdminArmEndpoint "<Administrative ARM endpoint>" -ServiceAdminCredentials $ServiceAdminCreds -TenantArmEndpoint "<Tenant ARM endpoint>" -TenantAdminCredentials $TenantAdminCreds -WindowsISOPath "<path where the WS2016 ISO is present>"
+```
+
+# To execute Canary as Tenant Administrator (if Windows Server 2016 or Windows Server 2012-R2 images are not present in PIR)
+```powershell
+# Download the WS2016 ISO image from: https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2016, and place it on your local machine
+# Install-Module AzureRM -RequiredVersion 1.2.8 -Force
```

Comment:
This should probably match what we have for Connect (no -Force and scoped to Current User) - and may soon need to be updated to reflect Profile support